app/modules/onboarding/assets/js/components/skip-button.js

```diff
@@ -18,8 +18,16 @@ export default function SkipButton( props ) {
 			if ( state.nextStep ) {
 				navigate( 'onboarding/' + state.nextStep );
 			}
-		},
-		action = button.action || skipStep;
+		};
+
+	const handleAction = () => {
+		if ( button.action ) {
+			button.action();
+			skipStep();
```

Comment:
only one Skip button in onboarding has 'action' functionality(it was added by me [here](https://github.com/elementor/elementor/pull/28193/files#diff-af7f9abf10af3580e741f8aaa756e4b347edf1b97a2112b097780573cd35f1a6R47)) - so we are safe to change the behavior